config/default.json

```diff
@@ -25,13 +25,16 @@
     "feeAndPayApi": "https://payment-api-prod.service.core-compute-prod.internal",
     "s2s": "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
   },
+  "userTimeoutInSeconds": 28800,
+  "iss": "https://forgerock-am.service.core-compute-idam-prod.internal:8443/openam/oauth2/hmcts",
```

Comment:
again we need to be careful here, confirm what prod url is returned

Reply:
@paddybasi  we are not going live with this feature now. I still need another refactor in the next sprint. But, we still need to find out whats the iss value